ci-operator/step-registry/ipi/install/rebalance-workloads/ipi-install-rebalance-workloads-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function rebalancePods() {
+  NS=$1
+  LABEL_SELECTOR=$2
+
+  NODE=$(oc get pods -n "${NS}" -l "${LABEL_SELECTOR}" -o jsonpath=\{'.items[*].spec.nodeName'\})
+
+  # If the workload is spread across more than 1 node, we don't need to rebalance its pods
+  NODE_COUNT=$(wc -w <<< "${NODE}")
+  if [ "${NODE_COUNT}" -ne 1 ]; then
+    echo "pods matching ${LABEL_SELECTOR} in namespace ${NS} are correctly spread"
+    return
+  fi
+
+  echo "need to rebalance pods matching ${LABEL_SELECTOR} in namespace ${NS}..."
+
+  oc adm cordon "${NODE}"
+
+  # Get all pods except one to keep it on the original node
+  PODS=$(oc get pods -n "${NS}" -l "${LABEL_SELECTOR}" -o jsonpath=\{'.items[*].metadata.name'\} | tr ' ' '\n' | head -n-1)
```

Comment:
This doesn't look right.  What if you select a terminated pod or a pod that is in a permanent containercreating state?